SoumyaSubhrajit/VehicleRentalHub
# Adding Docker file

# Pull Request: Adding Docker Configuration for Database

## Description
This pull request introduces Docker support for our application's database. The Docker configuration files are added to facilitate easier development, testing, and deployment.

## Changes Made
- Created a Dockerfile to define the environment for running our Node.js application.
- Added a docker-compose.yml file to orchestrate the setup of our Node.js app and MySQL database.
- Configured environment variables for connecting the application to the database in the Docker setup.

## How to Test
1. Ensure Docker is installed on your machine.
2. Clone this branch and navigate to the project directory.
3. Run the following command to build and start the Docker containers:
   ```bash
   docker-compose up --build
